stock_picking_invoice_link/models/stock_picking.py

```diff
@@ -17,9 +17,35 @@ class StockPicking(models.Model):
     invoice_id = fields.Many2one(
         comodel_name='account.invoice', string='Invoice',
         compute="_compute_invoice_id")
+    invoice_exists = fields.Boolean(
```

Comment:
It's for not adding another field to the model. I think we can live with the button leading to an empty invoice view.